### ISSUE TITLE
Publish Interactable?

### ISSUE THREAD

**Issue by Ashoat** [software-mansion/react-native-reanimated#1279]
## Description

Hey folks! So on my project we had to implement a `Swipeable` component. After trying some options (including the `Swipeable` in `react-native-gesture-handler`) we ended up finding that the `Interactable` inside `react-native-reanimated` was the best option. it has great performance characteristics, is very customizable, and uses Reanimated under the hood.

The only weird part - this `Interactable` is inside the Example folder, and isn't published with the package. It looks like `Interactable` is a reimplementation of [Wix's version]( https://github.com/wix/react-native-interactable/), which is a bit old and unmaintained, but still is pretty popular on [npm](https://www.npmjs.com/package/react-native-interactable). 

The version of `Interactable` inside `react-native-reanimated` is also popular enough to have its [own package](https://www.npmjs.com/package/react-native-interactable-reanimated). We would use it, but I'm worried it won't be maintained too well. instead, we're probably going to just directly include the entire `react-native-reanimated` Git repo in our `package.jso`n so we can get to the Example folder.

My request: would you guys consider moving your `Interactable` outside of the Example folder, and publishing it with the `react-native-reanimated` package? I think it's a very useful building block for complex interactions.

### Caveats

The big downside here would be that it would bloat package size, which might be particularly problematic given the recent work on `react-native-web` compatibility. An alternate approach would be to reorganize this repo as a monorepo, and separately publish the `Interactable`.

**Comment by nomi9995** [comment]
@Ashoat you can use this. it is from reanimated

```
yarn add react-native-interactable-reanimated
```

https://github.com/nomi9995/react-native-interactable-reanimated

**Comment by jkadamczyk** [comment]
Hi @Ashoat good to see you here.
We're sorry for replying this late, it's been over a year.
Our stance in terms of `Interactable` is we don't want to export it from Reanimated directly. I think in this case it is still something that is linked with Reanimated 1 and we are now fully focused on supporting and maintaining Reanimated 2.

If what @nomi9995 made is enough for you, great! If not I highly encourage you to just take what's in example and have it in your repo.

tl;dr We won't do it, hope you don't mind and found a way around it

Have a good and productive day!
